src/20231229_Multi_TreasuryManagementGSMFundingRWAStrategyPreparationsPart1/TreasuryManagementGSMFundingRWAStrategyPreparationsPart1.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Treasury Management - GSM Funding & RWA Strategy Preparations (Part 1)"
+author: "karpatkey_TokenLogic"
+discussions: "https://governance.aave.com/t/gho-stability-module-update/14442/10"
+---
+
+## Simple Summary
+
+In preparation for launching the GHO Stability Module (GSM) and the $1M RWA strategy with Centirfuge, this publication seeks to make available the necessary amount of USDC, USDT and DAI available on Ethereum.
+
+# Motivation
+
+The GSM is in the late stages of audit and is expected to be available for deployment in the near future. The Centrifuge proposal is currently mid way through the review process. The current stable coin holdings on Ethereum are insufficient to support both the GSM, RWA strategy and Service Provider (SP) commitments. Additional funds are needed on Ethereum to support the GSM.
```

Comment:
Remove "is in the late stages of audit and" as is not likely to be true anymore.